strr-platform-web/app/components/form/platform/ReviewConfirm.vue

```diff
@@ -46,6 +47,13 @@ onMounted(async () => {
   const validationResults = await Promise.all(validations)
   sectionErrors.value = validationResults.flatMap(result => result as MultiFormValidationResult)
 })
+
+// for BCeID use entered completingParty info, for BCSC use account info as per usual
+const completingPartyFullName = kcUser.value.loginSource === LoginSource.BCEID
+  ? `${platContactStore.completingParty?.firstName || '-'} ` +
```

Comment:
If no firstName, then is showing `-` part of the requirement?

Reply:
Yes, it is, Victor confirmed. Thanks